src/util.h

```diff
@@ -197,10 +197,15 @@ static SECP256K1_INLINE void memczero(void *s, size_t len, int flag) {
 /** If flag is true, set *r equal to *a; otherwise leave it. Constant-time.  Both *r and *a must be initialized and non-negative.*/
 static SECP256K1_INLINE void secp256k1_int_cmov(int *r, const int *a, int flag) {
     unsigned int mask0, mask1, r_masked, a_masked;
+    /* Access flag with a volatile-qualified lvalue.
+       This prevents clang from figuring out (after inlining) that flag can
+       take only be 0 or 1, which leads to variable time code. */
+    volatile int vflag = flag;
```

Comment:
you can make that `volatile unsigned int` and then remove the casting a few lines below?

Reply:
Hm, I don't see why this is better. I prefer the current form because then it's just copy'n'paste from above. (I know duplicated is to be avoided but duplicated *and* modified code is even worse.)